components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/latest/GitpodMetricControlProvider.kt

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package io.gitpod.jetbrains.remote.latest
+
+import com.jetbrains.ide.model.uiautomation.BeControl
+import com.jetbrains.ide.model.uiautomation.DefiniteProgress
+import com.jetbrains.rd.platform.codeWithMe.unattendedHost.metrics.providers.IdeMetricsProvider
+import com.jetbrains.rd.ui.bedsl.dsl.VerticalGridBuilder
+import com.jetbrains.rd.ui.bedsl.dsl.verticalGrid
+import com.jetbrains.rd.util.lifetime.Lifetime
+import com.jetbrains.rd.util.reactive.Property
+import com.jetbrains.rdserver.diagnostics.BackendDiagnosticsService
+import com.jetbrains.rdserver.unattendedHost.customization.controlCenter.performance.MetricControlProvider
+import com.jetbrains.rdserver.unattendedHost.customization.controlCenter.performance.createMetricProgressWithLabels
+import com.jetbrains.rdserver.unattendedHost.customization.controlCenter.performance.createProgressBar
+import com.jetbrains.rdserver.unattendedHost.customization.controlCenter.performance.createProgressRow
+
+class GitpodMetricControlProvider : MetricControlProvider {
+    override val id: String = "io.gitpod.jetbrains.remote"
+    override fun getControl(lifetime: Lifetime): BeControl {
+        return verticalGrid {
+            val backendDiagnosticsService = BackendDiagnosticsService.Companion.getInstance()
+           createCpuControl(this, backendDiagnosticsService, lifetime)
+            createMemoryControl(this, backendDiagnosticsService, lifetime)
+        }
+    }
+
+    private fun createCpuControl(ctx: VerticalGridBuilder, backendDiagnosticsService: BackendDiagnosticsService, lifetime: Lifetime) {
+        val data = createMetricProgressWithLabels(backendDiagnosticsService, lifetime, "gitpod_workspace_cpu_total", "gitpod_workspace_cpu_used", "gitpod_workspace_cpu_percentage")
```

Comment:
nice that we shod total and use for CPU as well, it would help to clarify for users how much CPU they actually have